desktop/apps/buy_art/stylesheets/index.styl

```diff
@@ -0,0 +1,350 @@
+  height 100%
+body
+  garamond()
+  font-size default-garamond-font-size
+  line-height 1.4em
+  text-rendering optimizeLegibility
+  -webkit-tap-highlight-color rgba(0, 0, 0, 0);
+a
+  color black
+  cursor pointer
+  -webkit-tap-highlight-color rgba(26, 26, 26, 0.301961);
+b, strong
+  font-weight bold
+em
+  font-style italic
+.small-caps
+  small-caps()
+input
+  garamond()
+  font-size default-font-size
+  appearance none
+  &[type='checkbox']
+    appearance checkbox
+  &[type='radio']
+    appearance radio
+  &:focus
+    outline none
+button, input
+  margin 0
+  outline none
+hr
+  position relative
+  height 1px
+  background-color gray-lighter-color
+  border none
+  margin 18px 0
+  clearfix()
+  &.is-invisible
+    background-color transparent
+sup
+  font-size 75%
+  line-height 1
+  vertical-align super
+sub
+  font-size 75%
+  line-height 1
+  vertical-align sub
+figure
+  margin 0
+
+#main-layout-container
+  margin-top (content-margin-top + main-header-height + welcome-header-height)
+
+// The 1250px max inner container
+.main-layout-container
+  max-width default-max-page-width
+  margin auto
+
+#blank-layout-container
+  height 100%
+
+// Scroll Frame styles
+#scroll-frame-spinner
+  position fixed
+  top 0
+  left 0
+  width 100%
+  height 100%
+  display none
+  z-index 10
+  background white
+.scroll-frame-loading #scroll-frame-spinner
+  display block
+.scroll-frame-open
+  overflow hidden
+
+#scripts
+  width 0
+  height 0
+
+.buy-art-buttons
+  width 270px
+  height 45px
+  padding 15px
+  box-sizing border-box
+  > i
+    margin-left 0
+
+.buy-art__svg-icon
+  display inline
+  vertical-align middle
+  margin-right 10px
+
+.buy-art-responsive-layout-container
+  .buy-art-main-layout-container
+    margin 0 auto
+    max-width 1250px
+  margin 0 20px
+  @media screen and (min-width: 768px)
+    margin 0 55px
+  @media screen and (min-width: 1440px)
+    margin 0 150px
+  @media screen and (min-width: 1660px)
+    margin 0 200px
+
+.buy-art-show-on-mobile
+  display none
+  @media screen and (max-width: 768px)
+    display inherit
+
+.buy-art-show-on-desktop
+  display inherit
+  @media screen and (max-width: 768px)
+    display none
+
+.buy-art-text-center-on-mobile
+  text-align inherit
+  @media screen and (max-width: 768px)
+    text-align center
+
+.buy-art-signup-button
+  avant-garde-size('body')
+  display inline-block
+  width 190px
+  height 45px
+  line-height 45px
+  background-color #6e1fff
+  color white
+  text-align center
+  text-decoration none
+
+.ba__email-signup-button
+  @extends .avant-garde-button
+  @extends .buy-art-buttons
+  margin-right 30px
+  background-color #ffffff
+.ba__facebook-signup-button
+  @extends .buy-art-buttons
+  background-color #586DB8
+  color white
+  avant-garde-size('body')
+  text-decoration none
+  line-height 1
+  > i
+    line-height 0
+    vertical-align -8px
+
+.buy-art-header
+  background-position center
+  background-repeat no-repeat
+  background-size cover
```

Comment:
Seeing these three rules repeated a bunch here, you can always make these into mixins:

```stylus
background-size-cover()
  background-position center
  background-repeat no-repeat
  background-size cover

.class
  +background-size-cover()
```